onnx/test/test_backend_onnxruntime.py

```diff
@@ -1,278 +1,279 @@
-# Copyright (c) ONNX Project Contributors
```

Comment:
this is frustrating - lintrunner did this. Otherwise I need to manually edit to make lint happy. Not sure if people working on Windows bearing the same pain?

Reply:
There should be an option in your editor for that. Check also git as it may do the replacement as well. On Windows, I use VSCode on WSL. I don't have this issue any more.

Reply:
fixed - vs code allows to set EOL